### PR TITLE
Moveconnect

### PR DESCRIPTION
**Pull request by hecon5**
I bit off more than I could chew at once with the connection class. 

Instead of having to keep rebasing every time Dev is updated, move these functions out of the `clsTableDef` and into `modSQLFunctions`, as it's more applicable to the SQL than just the tables. 

NOTE: I also removed the encryption/decryption from the connection build routines; it's my presumption that we'll excise the rest of the code prior to merging into `master` and releasing the next iteration.

### CODE REVIEW

**Comment by hecon5** [comment]
Nevermind; removed the encryption excision; I'll make a PR momentarily to do that.
Ready for comment @joyfullservice!

**Comment by joyfullservice** [comment]
I am kind of thinking these functions might belong more in `modSanitize`... In addition to SQL servers, the _connection string_ can connect to other data sources such as text files, ODBC connections, etc... The `modSqlFunctions` module is currently just used for ADP projects, and is specifically relating to Microsoft SQL server.

**Comment by hecon5** [comment]
Perhaps we should put them in `modConnect` then? I could see a few cases where it might be useful to have more connection routines than just sanitation, such as reporting the server/database name separately. 
This would actually lend itself to a class, but breaking it out first might be a good first step.

**Comment by joyfullservice** [comment]
Sure, `modConnect` sounds good to me.

**Comment by hecon5** [comment]
Done

File: Version Control.accda.src/modules/modConnect.bas [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                
+                ' Add all other sections
+                Case Else
+                    .Add strPart
+            End Select
+        Next lngPart
+        
+        ' Remove trailing semicolon, and return string
+        .Remove 1
+        SanitizeConnectionString = .GetStr
```
Comment (hecon5) [inline Version Control.accda.src/modules/modConnect.bas:63]: Is there a reason we're trimming the last semicolon? It seems to work fine either way, just curious.

Comment (joyfullservice) [reply Version Control.accda.src/modules/modConnect.bas:63]: The connection strings I was seeing in my system didn't have them. But like you say, it probably works fine either way. I was just trying to be consistent with the original.

Comment (hecon5) [reply Version Control.accda.src/modules/modConnect.bas:63]: If you're good with it, I think we should probably not remove it; there's an infinitesimally small chance that we could end up trimming a property setting, and also not altering the string will be faster than altering it (after testing, of course).

Comment (hecon5) [reply Version Control.accda.src/modules/modConnect.bas]: I removed it from here; it tests fine on my end; but verify it doesn't break anything.

**Pull request merged**